tests/playbooks/tests_states.yml

```diff
@@ -47,3 +47,16 @@
             state: down
     - include_tasks: tasks/assert-device_absent.yml
     - include_tasks: tasks/assert-profile_absent.yml
+
+    #downing non_existent profile
+    - name: Set down non_existent profile
```

Comment:
Please run this by yourself and check if it works. The CI also complains about it (Travis).

Reply:
It seems there was a linting error ( an extra blank line at the end ). I had checked if the playbook was running as expected .

Reply:
What does expected mean for you? Does it work or does it fail?

Reply:
It fails with this error message:
```
[WARNING]: [001] <error> #0, state:down persistent_state:absent,
'non_existent': Connection not found on system and 'type' not specified
fatal: [localhost]: FAILED! => {"changed": false, "msg": "error: Connection not found on system and 'type' not specified"}
```
which I think should happen as the connection does not exist and the role does not know that it is to be downed . Should something else happen ?

Reply:
This sounds correct. However, this means we cannot merge this as-is because it will break the CI. Therefore this also needs the fix for NM or some extra Ansible code that checks that this fails and ensures that the playbook will still succeed.